### PR TITLE
Eora Surcoats for Cleric and Templar

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Adds the Eoran Surcoat to the Cleric/Templar Eora worshippers.

Fixes the two neck uses of the Pestran Templar.

## Why It's Good For The Game

They have a surcoat sprite, why not use it? Plus a bugfix.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by Mogelix** [comment]
If I do not recall incorrectly, the person who added templars said that for lore reasons, (I don't know if supported by Guayo) there is no martial order of Eora representable as templars or clerics. Is that the case, authoritative lore people?

**Comment by sanshoom** [comment]
can we get pictures..

**Comment by AceVX** [comment]
Eroas prettiest Templar image

File: code/modules/jobs/job_types/roguetown/church/templar.dm [hunk 1]
```diff
@@ -43,10 +43,10 @@
 			cloak = /obj/item/clothing/cloak/stabard/templar/necra
 		if("Eora")
 			neck = /obj/item/clothing/neck/roguetown/psicross/eora
+			cloak = /obj/item/clothing/cloak/stabard/templar/eora
 			H.virginity = FALSE
 		if("Pestra")
 			neck = /obj/item/clothing/neck/roguetown/psicross/pestra
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/church/templar.dm:49]: I would've preferred to move the psycross to the wrist slot tbh

**Comment by SELFHELL** [comment]
Redundant for templars atleast since eorans arent on the allowed list.

**Review by ThePainkiller — APPROVED** [review]
Why does this amerit a testmerge, what even is there to merge or gather player input for?

**Pull request merged**